pkg/kubelet/token/token_manager.go

```diff
@@ -180,8 +181,8 @@ func (m *Manager) requiresRefresh(tr *authenticationv1.TokenRequest) bool {
 	if now.After(iat.Add(maxTTL)) {
```

Comment:
I would suggest:
1. Making maxJitter an attribute of the manager (so it can be disabled for unit tests if desired)
2. Initializing to 10*time.Second in NewManager
3. Computing jitter separately:

   ```go
   jitter := time.Duration(rand.Float64()*m.maxJitter)
   ```

4. Jittering both the maxTTL and the expiration:
   ```go
   if now.After(iat.Add(maxTTL-jitter)) {
   ...
   if now.After(exp.Add(-1*time.Duration((*tr.Spec.ExpirationSeconds*20)/100)*time.Second - jitter)) {
   ...
   ```